infrastructure/nomad-configuration/nomad-client-instance-user-data.tpl.sh

```diff
@@ -17,6 +17,20 @@
 # Change to home directory of the default user
 cd /home/ubuntu
 
+# Install, configure and launch our CloudWatch Logs agent
+cat <<EOF >awslogs.conf
+[general]
```

Comment:
What's this line do?

Reply:
We need an `awslogs.conf` file to be able to run `./awslogs-agent-setup.py` without any human interaction. This block create that file.